bin/acfg1.py

```diff
@@ -17,7 +17,7 @@
 METADATA_FILE_NAME = "acfg.metadata"
 
 
-def copy_tree(source, destination, dryrun=False):
+def copy_tree(source, destination, hostclasses=[], dryrun=False):
```

Comment:
Tuples are immutable, which is why it's the preferred alternative.  But yeah, mutable defaults are verboten in general.